src/v/archival/ntp_archiver_service.cc

```diff
@@ -568,13 +570,8 @@ ntp_archiver::upload_tx(upload_candidate candidate) {
 
     cloud_storage::tx_range_manifest manifest(path, tx_range);
 
-    // Note: tx-manifest is uploaded using 'remote::upload_manifest'
-    // method but it has the same tags as the segment because it has
-    // the same lifetime as corresponding segment and associated with
-    // the segment.
```

Comment:
nit: perhaps add a comment like this to `make_tx_manifest_tags`? Seems like useful context for an potentially surprising implementation.

Reply:
done